katello/hooks/pre/30-upgrade.rb

```diff
@@ -37,6 +37,17 @@ def migrate_foreman
   Kafo::Helpers.execute('foreman-rake db:migrate')
 end
 
+def postgresql_10_upgrade
+  #User should perform a backup
+  #Ensure there is at least 1x /var/lib/pgsql free disk space available on the main filesystem
+  Kafo::Helpers.execute('yum -y install rh-postgresql10-postgresql-server')
+  Kafo::Helpers.execute('foreman-maintain service stop')
+  Kafo::Helpers.execute('scl enable rh-postgresql10 -- postgresql-setup --upgrade')
+  Kafo::Helpers.execute('yum -y remove postgresql*')
```

Comment:
This also removes `postgresql-libs` which is a dependency of `tfm-rubygem-postgresql`, right? Probably also the `python3-psycopg2` package.